STYLEGUIDE.md

```diff
@@ -147,7 +220,7 @@ Default: https://google.github.io/styleguide/javascriptguide.xml
   });
   ```
 
-* <a name="js-operator-continuations"></a>
+- <a name="js-operator-continuations"></a>
```

Comment:
Why the change from `*` to `-` here when the rest of this bulleted list uses `*`?

Reply:
I was trying to make the formatting more consistent and changed all the `*`to `-`. But then I thought it wasn't needed in this PR and would take attention from the intended changes.